endpoint.go

```diff
@@ -612,6 +633,14 @@ func (ep *endpoint) setupDNS() error {
 	return ep.updateDNS(resolvConf)
 }
 
+// EndpointOptionPrimary function returns an option setter for making the
+// endpoint a primary endpoint
+func EndpointOptionPrimary() EndpointOption {
```

Comment:
Just programmatically alone and will be passed only for the implicit endpoint created when you do `docker run`. No need for UI/API changes